packages/executor/src/interfaces.ts

```diff
@@ -155,6 +155,8 @@ export interface NetworkConfig {
   userOpGasLimit: number; // 25kk by default
   bundleGasLimit: number; // 25kk by default
   kolibriAuthKey: string;
+  // catches EntryPoint v6 simulation reverts and returns them without revert
```

Comment:
Can we use the same forwarder for EP7 as well?

Reply:
yes, but EP7 doesn't revert on succesful simulation anyway, so it's not needed there